### PR TITLE
#869 docs: add examples and use cases for KONTENA_LB_CUSTOM_SETTINGS

### PR DESCRIPTION
**Pull request by giovannicandido**
Hi,
I would like to discus the follow documentation about KONTENA_LB_CUSTOM_SETTINGS

### CODE REVIEW

File: docs/using-kontena/loadbalancer.md [hunk 1]
```diff
@@ -241,3 +241,103 @@ These options are defined on the services that are balanced through Kontena Load
...
+```yaml
+environment:
+      KONTENA_LB_INTERNAL_PORT: 80
+      KONTENA_LB_VIRTUAL_HOSTS: www.kontena.io,kontena.io
+      KONTENA_LB_CUSTOM_SETTINGS: |
+        acl fonts_static path_end .eot .ttf .otf .woff
+        http-response add-header Access-Control-Allow-Origin "*" if fonts_static
+```
+
+**Wrong**:
```
Comment (jnummelin) [inline docs/using-kontena/loadbalancer.md]: Not sure if we need the **wrong** example

File: docs/using-kontena/loadbalancer.md [hunk 2]
```diff
@@ -241,3 +241,103 @@ These options are defined on the services that are balanced through Kontena Load
...
+    environment:
+      KONTENA_LB_INTERNAL_PORT: 80
+      KONTENA_LB_VIRTUAL_HOSTS: www.kontena.io,kontena.io
+      KONTENA_LB_CUSTOM_SETTINGS: |
+        acl fonts_static path_end .eot .ttf .otf .woff
+        http-response add-header Access-Control-Allow-Origin "*" if fonts_static
+      
+```
+
+Docs: https://www.haproxy.com/doc/aloha/7.0/haproxy/http_rewriting.html#rewriting-http-responses
```
Comment (jnummelin) [inline docs/using-kontena/loadbalancer.md]: The doc references should be made to the open source version as Kontena LB is based on that and not the commercial Aloha version.

File: docs/using-kontena/loadbalancer.md (hunk 1 above)
Comment (giovannicandido) [inline docs/using-kontena/loadbalancer.md]: Agreed, this could be confusing for the reader.

File: docs/using-kontena/loadbalancer.md (hunk 2 above)
Comment (giovannicandido) [inline docs/using-kontena/loadbalancer.md]: I didn't pay attention for that when I made the google search. Thanks!

**Comment by SpComb** [comment]
The `docs` have been moved to https://github.com/kontena/docs, any docs PRs need to be re-submitted there.

**Pull request closed without merging**